jobbergate-api/jobbergate_api/apps/services.py

```diff
@@ -585,15 +575,14 @@ async def _get_file_data_from_url(self, file_url: AnyUrl) -> io.BytesIO:
                     raise_exc_class=ServiceError,
                     raise_kwargs=dict(status_code=status.HTTP_400_BAD_REQUEST),
                 )
-
-                session = Session()
-                async with session.resource("s3", url_string) as s3:
-                    bucket = await s3.Bucket(bucket_name)
-                    s3_obj = await bucket.Object(key)
-                    await s3_obj.download_fileobj(file_obj)
+                file_url_string = f"https://{bucket_name}.amazonaws.com{key}"
```

Comment:
we should somehow verify for a trailing slash here in the `key` variable

Reply:
We are already enforcing that it's defined. If defined, the path will always have a leading slash due to the way `AnyUrl` parses them.

Reply:
right. Also regarding this line, as mentioned [here](https://app.clickup.com/t/18022949/PENG-2426?comment=90110089671267&threadedComment=90110090042093), it is missing `.s3` before `.amazonaws`

Reply:
> We are already enforcing that it's defined. If defined, the path will always have a leading slash due to the way `AnyUrl` parses them.

but I actually meant a trailing slash, not a leading slash

Reply:
It's not a valid s3 url if it has a trailing slash, is it? Are you proposing that we strip trailing slashes?

Reply:
I'm actually proposing we don't accept URLs with trailing slash

Reply:
just noticed you're calling `raise_for_status`. Disregard the conversation.